rules/linux/credential_access_potential_successful_linux_ftp_bruteforce.toml

```diff
@@ -22,19 +22,34 @@ language = "eql"
-```
-Kibana -->
-Management -->
-Integrations -->
-Auditd Manager -->
-Add Auditd Manager
-```
-`Auditd_manager` subscribes to the kernel and receives events as they occur without any additional configuration. However, if more advanced configuration is required to detect specific behavior, audit rules can be added to the integration in either the "audit rules" configuration box or the "auditd rule files" box by specifying a file to read the audit rules from. 
-```
-For this detection rule no additional audit rules are required to be added to the integration. 
-```
-Add the newly installed `auditd manager` to an agent policy, and deploy the agent on a Linux system from which auditd log files are desirable.
+
+This rule requires the use of the `auditd_manager` integration.
+The Auditd Manager Integration receives audit events from the Linux Audit Framework that is a part of the Linux kernel.
+Auditd Manager provides a user-friendly interface and automation capabilities for configuring and monitoring system
+auditing through the auditd daemon. With `auditd_manager`, administrators can easily define audit rules,
+track system events, and generate comprehensive audit reports, improving overall security and compliance in the system.
+
+The following steps should be executed in order to add the
+Elastic Agent System integration "auditd_manager" on a Linux System.
+
+- Go to the Kibana home page and click Add integrations.
+- In the query bar, search for Auditd Manager and select the integration to see more details about it.
+- Click Add Auditd Manager
+- Configure the integration name and optionally add a description.
+- Review optional and advanced settings accordingly.
+- Add the newly installed `auditd manager` to an existing or a new agent policy, and deploy the agent on a Linux system
+  from which auditd log files are desirable.
+- Click Save and Continue.
+
+Auditd Manager subscribes to the kernel and receives events as they occur without any additional configuration.
+However, if more advanced configuration is required to detect specific behavior, audit rules can be added
+to the integration in either the "audit rules" configuration box or the "auditd rule files" box by specifying a file
+to read the audit rules from.
+For more details on the integeration refer - https://docs.elastic.co/integrations/auditd_manager
+
+Rule Specific Setup Note
+- For this detection rule no additional audit rules are required to be added to the integration.
+
```

Comment:
For auditbeat documentation please visit https://www.elastic.co/guide/en/beats/auditbeat/current/auditbeat-overview.html .